instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/ui/routeline/RouteLineLayersTest.kt

```diff
@@ -372,6 +385,59 @@ class RouteLineLayersTest : BaseTest<BasicNavigationViewActivity>(
         countDownLatch.await()
     }
 
+    /**
+     * This test ensures that we're not crashing when parsing alternative routes metadata
+     * in a specific case (caught by crashlytics) that involved
```

Comment:
Did you find the route which caused the crash? Just wondering where the input test data came from.

Reply:
I took the routes from our backend logs based on the UUID of a similar crash caught in 1TAP.